### ISSUE TITLE
Cann't get logs of Postgres by docker logs

### ISSUE THREAD

**Issue by jxl4650152** [bitnami/bitnami-docker-postgresql-repmgr#32]
**Description**

Only Repmgr logs are showed in `docker logs` or `Kubectl logs`, no Postgres log is output.
**Steps to reproduce the issue:**
`docker logs pgha-postgresql-ha-postgresql-0`
**Describe the results you received:**
```postgresql-repmgr 03:24:25.62
postgresql-repmgr 03:24:25.62 Welcome to the Bitnami postgresql-repmgr container
postgresql-repmgr 03:24:25.62 Subscribe to project updates by watching https://github.com/bitnami/bitnami-docker-postgresql-repmgr
postgresql-repmgr 03:24:25.63 Submit issues and feature requests at https://github.com/bitnami/bitnami-docker-postgresql-repmgr/issues
postgresql-repmgr 03:24:25.63
postgresql-repmgr 03:24:25.64 INFO  ==> ** Starting PostgreSQL with Replication Manager setup **
postgresql-repmgr 03:24:25.66 INFO  ==> Validating settings in REPMGR_* env vars...
postgresql-repmgr 03:24:25.66 INFO  ==> Validating settings in POSTGRESQL_* env vars..
postgresql-repmgr 03:24:25.67 INFO  ==> Querying all partner nodes for common upstream node...
postgresql-repmgr 03:24:25.75 INFO  ==> There are no nodes with primary role. Assuming the primary role...
postgresql-repmgr 03:24:25.76 INFO  ==> Preparing PostgreSQL configuration...
postgresql-repmgr 03:24:25.77 INFO  ==> postgresql.conf file not detected. Generating it...
postgresql-repmgr 03:24:25.85 INFO  ==> Preparing repmgr configuration...
postgresql-repmgr 03:24:25.86 INFO  ==> Initializing Repmgr...
postgresql-repmgr 03:24:25.86 INFO  ==> Initializing PostgreSQL database...
postgresql-repmgr 03:24:25.87 INFO  ==> Custom configuration /opt/bitnami/postgresql/conf/postgresql.conf detected
postgresql-repmgr 03:24:25.87 INFO  ==> Custom configuration /opt/bitnami/postgresql/conf/pg_hba.conf detected
postgresql-repmgr 03:24:49.67 INFO  ==> Starting PostgreSQL in background...
postgresql-repmgr 03:24:50.41 INFO  ==> Changing password of postgres
postgresql-repmgr 03:24:50.44 INFO  ==> Creating replication user repmgr
postgresql-repmgr 03:24:50.48 INFO  ==> Configuring replication parameters
postgresql-repmgr 03:24:50.50 INFO  ==> Configuring fsync
postgresql-repmgr 03:24:50.51 INFO  ==> Starting PostgreSQL in background...
postgresql-repmgr 03:24:50.52 INFO  ==> Creating repmgr user: repmgr
postgresql-repmgr 03:24:50.58 INFO  ==> Creating repmgr database: repmgr
postgresql-repmgr 03:24:55.34 INFO  ==> Stopping PostgreSQL...
postgresql-repmgr 03:24:56.35 INFO  ==> Starting PostgreSQL in background...
postgresql-repmgr 03:24:56.54 INFO  ==> Registering Primary...
postgresql-repmgr 03:24:56.82 INFO  ==> Loading custom scripts...
postgresql-repmgr 03:24:56.83 INFO  ==> Stopping PostgreSQL...
postgresql-repmgr 03:24:57.84 INFO  ==> ** PostgreSQL with Replication Manager setup finished! **

postgresql-repmgr 03:24:57.86 INFO  ==> Starting PostgreSQL in background...
postgresql-repmgr 03:24:58.29 INFO  ==> ** Starting repmgrd **
[2020-08-28 03:24:58] [NOTICE] repmgrd (repmgrd 5.1.0) starting up
INFO:  set_repmgrd_pid(): provided pidfile is /opt/bitnami/repmgr/tmp/repmgr.pid
[2020-08-28 03:24:58] [NOTICE] starting monitoring of node "pgha-postgresql-ha-postgresql-0" (ID: 1000)
[2020-08-28 03:24:58] [NOTICE] monitoring cluster primary "pgha-postgresql-ha-postgresql-0" (ID: 1000)
[2020-08-28 03:25:28] [NOTICE] new standby "pgha-postgresql-ha-postgresql-1" (ID: 1001) has connected
[2020-08-28 03:26:16] [NOTICE] new standby "pgha-postgresql-ha-postgresql-2" (ID: 1002) has connected
```

**Describe the results you expected:**
Show Postgres log and repmgr log

**Additional information you deem important (e.g. issue happens only occasionally):**

Default configuration
**Version**
11.8.0-debian-10-r61
- Output of `docker version`:

```
(paste your output here)
```

- Output of `docker info`:

```
(paste your output here)
```

- Output of `docker-compose version` (if applicable):

```
(paste your output here)
```

**Additional environment details (AWS, VirtualBox, Docker for MAC, physical, etc.):**

**Comment by javsalgar** [comment]
Hi,

Thanks for the input. I will open a task for evaluation. I cannot guarantee an ETA but we will update the ticket as soon as we have more news

**Comment by jxl4650152** [comment]
Thanks. 
I figured out that I need to enable `.Values.postgresqlImage.debug` to get the logs of postgresql.
Btw, the comments about `.Values.postgresqlImage.debug` is incorrect and misleading. 
```bash
## Set to true if you would like to see extra information on logs
## ref:  https://github.com/bitnami/minideb-extras/#turn-on-bash-debugging
##
debug: true
```
The ref in the comments is about turning bash debugging.

**Comment by javsalgar** [comment]
Hi,

Yes, that's a possible workaround. However, we are working on showing the PostgreSQL logs without having to set `debug = true`, because it is meant for more debugging on the bash initialization.

**Comment by jxl4650152** [comment]
Thanks.

**Comment by FraPazGal** [comment]
Hi @jxl4650152,

Sorry for the late response, I looked at this issue last week but couldn't work on it sooner. Like javsalgar said, we wanted to avoid having to set `debug = true`, as we think that outputing the PostgreSQL logs should be the natural behavior. As such, we have enabled by default the PosgreSQL logs of the database from each node. The change was included in today's release, so the container should be now showing both the Postres and repmgr logs.

Thanks for making us aware of this issue and your patience.
